### ISSUE TITLE
л{й}я = н{й}я

### ISSUE THREAD

**Issue by hectoralos** [apertium/apertium-chv#27]
In words finishing in `н{й}я` `й` is substituted by `ь` before `ӑ`. This is correct. The same should happen for words finishing in `л{й}я`. For instance:

```
$ echo "Коля<np><ant><m><ins>" | hfst-lookup .deps/chv.LR.lexc.hfst 2>/dev/null
Коля<np><ant><m><ins>	Кол{й}я>п{A}	0,000000
```
Generates `Колйӑпа` instead of `Кольӑпа`.

**Comment by ftyers** [comment]
```
$ echo "Коля<np><ant><m><ins>" | hfst-lookup .deps/chv.LR.hfst 
Коля<np><ant><m><ins>	Кольӑпа	0,000000
```

Should be fixed in 23fbb91 .

**Comment by hectoralos** [comment]
For the record, I have seen that the change is general. This almost relates to Russian personal names like Катя, Настя, Вася, etc. There very few words like сакля, to which to rule applies.
